Assets/Scripts/Services/History/RoutesFiles/RoutesReader.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Model;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.AccessControl;
+using System.Text;
+using UnityEngine;
+
+namespace Services.RoutesFiles {
+	public class RoutesReader : IDisposable {
+		private List<StreamReader> streams = new List<StreamReader>();
+
+		private const string DATA_FILE_EXTENSION = ".wgroute";
+		private string routesPath;
+		private string[] fileNames;
+		private int[] routeLength;
+
+		public RoutesReader(string path, string dataFilePostfix = "") {
```

Comment:
why do we have this `dataFilePostfix` param that start all the way in `HistoryService`? Remove it all if not used